### PR TITLE
Deep link

### PR DESCRIPTION
**Pull request by anastasiaShpakava**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,6 +25,17 @@
...
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="template.softteco.com.deep.link" />
+                <data android:scheme="http" />
+                <data android:host="template.softteco.com.deep.link" />
```
Comment (EvgeniyFilippov) [inline app/src/main/AndroidManifest.xml:37]: can be simplified:

 <data android:scheme="http" android:host="template.softteco.com.deep.link" />
 <data android:scheme="https" android:host="template.softteco.com.deep.link" />

**Review by EvgeniyFilippov — APPROVED** [review]

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -25,6 +25,16 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
```
Comment (bstolyarov) [inline app/src/main/AndroidManifest.xml:30]: Empty line

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -25,6 +25,16 @@
...
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
```
Comment (bstolyarov) [inline app/src/main/AndroidManifest.xml:33]: Empty line

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -25,6 +25,16 @@
```
Comment (bstolyarov) [inline app/src/main/AndroidManifest.xml:25]: Empty line

**Review by bstolyarov — COMMENTED** [review]
Looks good, just remove empty line

**Review by bstolyarov — APPROVED** [review]

**Pull request merged**